### PR TITLE
added v2 for github refresh job

### PR DESCRIPTION
**Pull request by yzainee-zz**
Changes added to accommodate v2 stack report in the github refresh job

### CODE REVIEW

File: src/github_refresh.py [hunk 1]
```diff
@@ -78,27 +82,46 @@ def get_epv_list():
...
-                        "ecosystem": eco,
-                        "name": name
-                    })
+                    epv_list[eco].append(name)
             else:
                 logger.info("No deps found in the report for the ecosystem {}".format(eco))
+
+    logger.info("Fetching venus V2 stack report for the date {}".format(yest_date))
+    v2_stack_json = retrieve_dict("v2/daily/" + yest_date + ".json")
+    if v2_stack_json:
```
Comment (prashbnair) [inline src/github_refresh.py:79]: Common code across v1 and v2 , can be moved into a common method.

File: src/github_refresh.py [hunk 2]
```diff
@@ -78,27 +82,46 @@ def get_epv_list():
...
+        for eco in eco_list:
+            dep_list = v2_stack_json.get('stacks_summary', {}).get(eco, {}) \
+                .get('unique_dependencies_with_frequency', {})
+            if bool(dep_list):
+                for k in dep_list:
+                    name = k.split(" ")[0]
+                    epv_list[eco].append(name)
+            else:
+                logger.info("No deps found in the report for the ecosystem {}".format(eco))
+    epv_list = {
```
Comment (prashbnair) [inline src/github_refresh.py]: Why is this required, is it to remove duplicates?

File: qa/runtests.sh [hunk 3]
```diff
@@ -16,22 +16,21 @@ check_python_version() {
...
 function prepare_venv() {
-        # we want tests to run on python3.6
-        printf 'checking alias `python3.6` ... ' >&2
-        PYTHON=$(which python3.6 2> /dev/null)
-        if [ "$?" -ne "0" ]; then
-                printf "%sNOT FOUND%s\n" "${YELLOW}" "${NORMAL}" >&2
-
-                printf 'checking alias `python3` ... ' >&2
-                PYTHON=$(which python3 2> /dev/null)
+        VIRTUALENV=$(which virtualenv)
```
Comment (prashbnair) [inline qa/runtests.sh:19]: Do we want to use a separate virtualenv command, I do not use it. I use the python -m command.

**Review by prashbnair — CHANGES_REQUESTED** [review]

File: src/github_refresh.py (hunk 1 above)
Comment (yzainee-zz) [reply src/github_refresh.py:79]: yes i know.. but very soon i will be removing v1 all together as there wont be any data coming in v1. let me know if you still want me to write the common stuff

File: src/github_refresh.py (hunk 2 above)
Comment (yzainee-zz) [reply src/github_refresh.py]: yes

File: qa/runtests.sh (hunk 3 above)
Comment (yzainee-zz) [reply qa/runtests.sh:19]: This was a standard that pavel had set for all the runtest files. i think he missed out on this one.

File: src/github_refresh.py (hunk 1 above)
Comment (yzainee-zz) [reply src/github_refresh.py:79]: i kept it simple so that deletion of code is easy, in some days from now

File: src/github_refresh.py (hunk 2 above)
Comment (prashbnair) [reply src/github_refresh.py]: Can you not create a set in the first place and try adding to it.

File: src/github_refresh.py (hunk 1 above)
Comment (prashbnair) [reply src/github_refresh.py:79]: My suggestion is to create a common method. I do not know when we will get time to clean up V1 code.

**Comment by yzainee-zz** [comment]
@prashbnair PTAL.

**Review by prashbnair — APPROVED** [review]
